### PR TITLE
Add TDD exercise for CLI prompt

### PR DESCRIPTION
**Pull request by JoeShep**
I felt a need for an exercise to allow students to loop back to TDD, as well as play with prompting input, before diving into sprint 2, which uses both skills/tools. Feedback on level of difficulty and clarity of requirements, please
@Rian501

### CODE REVIEW

**Review by Rian501 — COMMENTED** [review]
I like it, especially telling them to just start with two methods and build out from there. The reqs seem fairly clear to me, but as usual they should ask about unclarity.

File: post-orientation-exercises/TDD/prompt-training.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
++ start_date
++ end_date
++ course_category
+
+The app should enable the following actions:
+GET all/one course
+DELETE a course
+POST a course
+
+Use test-driven development to buld that logic of your app ( Do not test the UI functionality). Remember to practice `red green refactor` and write no feature before the test for that feature, and only write enough code in each iteraction to make the test pass.
```
Comment (Rian501) [inline post-orientation-exercises/TDD/prompt-training.md]: typo: build

**Comment by JoeShep** [comment]
I'm unclear on whether "unclarity" is a word. 😏

File: post-orientation-exercises/TDD/prompt-training.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+The app should enable the following actions:
+GET all/one course
+DELETE a course
+POST a course
+
+Use test-driven development to build that logic of your app ( Do not test the UI functionality). Remember to practice `red green refactor` and write no feature before the test for that feature, and only write enough code in each iteraction to make the test pass.
+
+### The UI
+Use the [prompt npm package](https://www.npmjs.com/package/prompt)
+to facilitate interaction with the user on te command line. You can use the [colors](https://www.npmjs.com/package/colors) package to spice up the terminal view
```
Comment (Rian501) [inline post-orientation-exercises/TDD/prompt-training.md]: typo the

**Pull request merged**